cli/src/main/java/com/liferay/blade/cli/command/InitCommand.java

```diff
@@ -162,7 +162,7 @@ public void execute() throws Exception {
 		else {
 			WorkspaceProvider workspaceProvider = bladeCLI.getWorkspaceProvider(baseDir);
 
-			if (workspaceProvider != null) {
+			if ((workspaceProvider != null) || bladeCLI.isInWorkspace(baseDir)) {
```

Comment:
I'm not sure if this is the best place to put this check. I put it here to take advantage of the fact that the error was already being displayed here. However, I also considered the possibility of checking this earlier (as early as just after the `destDir.exists()` check in L105).

What do you think would be a better approach?